game/src/test/scala/io/aigar/controller/AigarStackSpec.scala

```diff
@@ -0,0 +1,38 @@
+import io.aigar.controller._
+import io.aigar.controller.response.{ErrorResponse}
+
+import org.json4s.{DefaultFormats, Formats}
+import org.json4s.jackson.JsonMethods._
+
+import org.scalatra.test.specs2._
+import org.specs2.matcher._
+
+object ConcreteStack extends AigarStack {
+  get("/:id") {
```

Comment:
Could we rename `id` to `status` or similar? :) It got me a bit confused at first.